packages/amplify_authenticator/lib/src/widgets/authenticator_input_config.dart

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+*/
+
+import 'package:amplify_authenticator/amplify_authenticator.dart';
+
+/// Configuration for defining content of radio and switch types in [AuthenticatorFormField].
+class SelectableConfig<T> {
+  final List<InputSelection> selections;
+  late T? selectionValue;
+  SelectableConfig({required this.selections});
```

Comment:
`const SelectableConfig`

Reply:
Can you do that without make selectionValue final?